<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HashFactoryUI</class>
 <widget class="QMainWindow" name="HashFactoryUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hash Factory</string>
  </property>
  <property name="windowIcon">
   <iconset resource="build/Desktop-Release/qmake_qmake_immediate.qrc">
    <normaloff>:/images/security.png</normaloff>:/images/security.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Check New File</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>File Path:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="pathFileLineEdit">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="browseButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="build/Desktop-Release/qmake_qmake_immediate.qrc">
             <normaloff>:/images/folder_open.png</normaloff>:/images/folder_open.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Checker</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="checkerComboBox">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>SHA-256</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>SHA-1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>MD5</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="calculateButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Calculate</string>
           </property>
           <property name="icon">
            <iconset resource="build/Desktop-Release/qmake_qmake_immediate.qrc">
             <normaloff>:/images/calculate.png</normaloff>:/images/calculate.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Hash</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="calculatedHashFile">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="typeCombo">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>trusted</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>malware</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>suspect</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addToDB">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Add </string>
           </property>
           <property name="icon">
            <iconset resource="build/Desktop-Release/qmake_qmake_immediate.qrc">
             <normaloff>:/images/database_upload.png</normaloff>:/images/database_upload.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Hash Factory</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="loadTable">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Load</string>
           </property>
           <property name="icon">
            <iconset resource="build/Desktop-Release/qmake_qmake_immediate.qrc">
             <normaloff>:/images/database_search.png</normaloff>:/images/database_search.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="filterSearch">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="columnFilterBox">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>FileName</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Checker</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Hash</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="filterButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="build/Desktop-Release/qmake_qmake_immediate.qrc">
             <normaloff>:/images/search.png</normaloff>:/images/search.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <column>
          <property name="text">
           <string>Operations</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>FileName</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Type</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Checker</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Hash</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Creation Date</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="exportButton">
           <property name="text">
            <string>Export </string>
           </property>
           <property name="icon">
            <iconset resource="build/Desktop-Release/qmake_qmake_immediate.qrc">
             <normaloff>:/images/file_export.png</normaloff>:/images/file_export.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="importButton">
           <property name="text">
            <string>Import</string>
           </property>
           <property name="icon">
            <iconset resource="build/Desktop-Release/qmake_qmake_immediate.qrc">
             <normaloff>:/images/file_open.png</normaloff>:/images/file_open.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>656</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="build/Desktop-Release/qmake_qmake_immediate.qrc">
     <normaloff>:/images/info.png</normaloff>:/images/info.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="build/Desktop-Release/qmake_qmake_immediate.qrc">
     <normaloff>:/images/help.png</normaloff>:/images/help.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="build/Desktop-Release/qmake_qmake_immediate.qrc">
     <normaloff>:/images/exit.png</normaloff>:/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="build/Desktop-Release/qmake_qmake_immediate.qrc"/>
 </resources>
 <connections/>
</ui>
